app/models/user.rb

```diff
@@ -8,6 +10,19 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
```suggestion
    update_column :remember_digest, User.digest(remember_token)
```
Dùng thèn này nhé, hiệu suất tốt hơn, vì chỗ này ko cần chạy callback nè